README.md

```diff
@@ -459,8 +483,10 @@ September, 2005
 Secure RTP is defined in [RFC 3711](https://www.ietf.org/rfc/rfc3711.txt).
 The counter mode definition is in Section 4.1.1.
 
-SHA-1 is defined in FIPS-180-1, available online at the NIST website.
+SHA-1 is defined in [RFC 3174](https://www.ietf.org/rfc/rfc3174.txt).
```

Comment:
We could reference that document, but it's really a copy of FIPS 180-1 (and it admits to such).  The current SHA-1 spec is in FIPS 180-4 (http://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.180-4.pdf).  That would be the most proper reference, I think, but it will be FIPS 180-5 the next time it is revised.